Makefile

```diff
@@ -0,0 +1,15 @@
+install:
+	@echo "--- 🚀 Installing project dependencies ---"
+	pip install -e .
```

Comment:
This will only install `browsergym` as a meta-package, the dependencies (core, webarena, workarena) will be fetched from pypi.

This should work
```sh
pip install -e core/
pip install -e miniwob/
pip install -e webarena/
```